src/main/java/org/assertj/vavr/internal/Maps.java

```diff
@@ -98,6 +99,31 @@ public static Maps instance() {
         }
     }
 
+    /**
+     * Verifies that the actual {@code Map} contains the given keys.
+     *
+     * @param <K>    key type
+     * @param <V>    value type
+     * @param info   contains information about the assertion.
+     * @param actual the given {@code Map}.
+     * @param keys   the given keys
+     * @throws NullPointerException     if the array of keys is {@code null}.
+     * @throws IllegalArgumentException if the array of keys is empty.
+     * @throws AssertionError           if the given {@code Map} is {@code null}.
+     * @throws AssertionError           if the given {@code Map} does not contain the given keys.
+     */
```

Comment:
Some sentences above end with `.` whereas other do not.

Reply:
👍